README.md

```diff
@@ -75,34 +75,143 @@ However, there might be cases in which not-yet released demos or features are on
 * Do not forget to source `setup.bash` from the `install` folder!
 
 
-# Getting Started with ros2_control
+# Getting Started with demos
 
-Each of the described example cases from the [roadmap](https://github.com/ros-controls/roadmap/blob/master/design_drafts/components_architecture_and_urdf_examples.md) has its own launch and URDF file.
-
-## Starting example robots
-
-Each example is started with a single launch file which starts up the robot hardware, loads controller configurations and it also opens `rviz2`.
+This repository provides two simple example robots: a 2 degrees of freedom manipulator - *RRBot* - and a mobile differential drive base - *DiffBot*.
 
-The `rviz2` setup can be recreated following these steps:
+The first two examples demonstrate the minimal setup for those two robots to run.
+Later examples show more details about `ros2_control`-concepts and some more advanced use-cases.
 
-- The robot models can be visualized using `RobotModel` display using `/robot_description` topic.
-- Or you can simply open the configuration from `rviz` folder in `rrbot_description` package manually or directly by executing:
-  ```
-  rviz2 --display-config `ros2 pkg prefix rrbot_description`/share/rrbot_description/config/rrbot.rviz
-  ```
+## *RRBot*
 
 *RRBot*, or ''Revolute-Revolute Manipulator Robot'', is a simple 3-linkage, 2-joint arm that we will use to demonstrate various features.
 It is essentially a double inverted pendulum and demonstrates some fun control concepts within a simulator and was originally introduced for Gazebo tutorials.
 The *RRBot* URDF files can be found in the `urdf` folder of `rrbot_description` package.
 
+1. To check that *RRBot* descriptions are working properly use following launch commands:
+
+   *RRBot*
+   ```
+   ros2 launch rrbot_description view_robot.launch.py
+   ```
+   **NOTE**: Getting the following output in terminal is OK: `Warning: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist`.
+             This happens because `joint_state_publisher_gui` node need some time to start.
```

Comment:
```suggestion
             This happens because `joint_state_publisher_gui` node need some time to start.

The `joint_state_publisher_gui` provides a GUI to generate  a random configuration for rrbot. It is immediately displayed in `Rviz`.
```